src/main/groovy/io/saagie/plugin/dataops/utils/SaagieUtils.groovy

```diff
@@ -1249,6 +1249,25 @@ class SaagieUtils {
 		outputStream.close()
 		inputStream.close()
 	}
+	// This function is helpful to get for example size of a file without downloading it
+	
+	int getOnlyHeaderInformationFromApi(String url, OkHttpClient client, version) {
+		def response = null;
+		try {
+			logger.debug("getting file size from header for v1 ${version}....")
+			Request request = this.buildRequestForFileHeader(url)
+			response = client.newCall(request).execute()
+			return response.body().contentLength();
+		} catch (IOException e) {
+			if (response!=null) {
+				response.close();
+				
+			}
+			e.printStackTrace();
```

Comment:
Should we keep that ? I think the error should go to an error or debug statement